### PR TITLE
[sc-5119]  Bug overview page stays loading infinitely

### PR DESCRIPTION
**Pull request by jakubswierczek**
# [Bug on overview page](https://app.shortcut.com/oazo-apps/story/5119/bug-overview-page-stays-loading-infinitely)
We introduced a regression on the overview page. Loading was infinite. 
  
## Changes 👷‍♀️
* Top Assets and position have timeout - 10 sec. After that time the component will not show. 
* Requests for cache for positions are combined in one request. 
  
## How to test 🧪
* Go to the overview page

### CODE REVIEW

File: features/vaultsOverview/pipes/positionsOverviewSummary.ts [hunk 1]
```diff
@@ -168,5 +169,7 @@ export function createPositionsOverviewSummary$(
     })),
     debounceTime(500),
     distinctUntilChanged(isEqual),
+    timeout(1000 * 10),
+    catchError(() => of({ assetsAndPositions: [], percentageOther: zero, totalValueUsd: zero })),
```
Comment (sennett) [inline features/vaultsOverview/pipes/positionsOverviewSummary.ts:173]: Not for this PR, but we should talk more with other fullstacks about swallowing errors like this.

File: features/vaultHistory/vaultHistory.ts [hunk 2]
```diff
@@ -243,6 +242,28 @@ function addReclaimFlag(events: VaultHistoryEvent[]) {
   })
 }
 
+export function flatEvents([events, automationEvents]: [
```
[the remark this answers is not in the document]
Comment (jakubswierczek) [reply features/vaultHistory/vaultHistory.ts:245]: yes, we used it in `vaultsHistory.ts`

[the remark this answers is not in the document]
Comment (sennett) [reply features/vaultHistory/vaultHistory.ts:245]: That's this file?

[the remark this answers is not in the document]
Comment (jakubswierczek) [reply features/vaultHistory/vaultHistory.ts:245]: no, this is `vaultHistory` -> for one vault. 
I added a pipeline for a couple of vaults. I don't have idea how to name that file ;)

**Review by sennett — APPROVED** [review]

**Pull request merged**